src/node/channels.h

```diff
@@ -20,13 +20,13 @@
 
 namespace ccf
 {
-  using SeqNo = uint64_t;
-  using GcmHdr = crypto::GcmHeader<sizeof(SeqNo)>;
+  using SendNonce = uint64_t;
```

Comment:
This is actually used like a sequence number (number of messages since last key rotation, which is limited to 2^24.5 now). Not sure we need a type for this at all.

Reply:
I'm just renaming the existing type so it doesn't squat on `ccf::SeqNo` for now. I agree it may be simpler to remove this if we don't have a clean name for it, though in general I think its a good idea to add name for these simple types whenever its used for a distinct purpose (to reduce some errors where they're used in the wrong place, or at least make it possible to unpick the mistake after the fact).